hs_core/templates/pages/genericresource.html

```diff
@@ -830,14 +830,13 @@ <h4 class="modal-title">Manage access</h4>
 
         {#        ======= Content =======#}
 
+        {% if cm.resource_type != "ToolResource" %}
```

Comment:
OK. +1 then. Unfortunately this will conflict with develop branch now that the PR #862 has been merged in. Sorry you will have to fix the conflict with latest develop now.